oracle/oracle.go

```diff
@@ -1000,35 +1000,51 @@ func (o *oracleOps) Enumerate(volumeIds []*string,
 	for _, volIds := range volumeIds {
 		volIDsMap[*volIds] = *volIds
 	}
-	for _, vol := range resp.Items {
-		_, ok := volIDsMap[*vol.Id]
-		if !ok {
-			continue
-		}
+	for {
```

Comment:
nit: You can make the first ListVolumes call also inside the for loop. A slightly optimized way to write this could be - 
```
var nextPage *string
for {
        req := core.ListVolumesRequest{
		CompartmentId: common.String(o.compartmentID),
	}
        if nextPage != nil {
                 req.Page = nextPage
        }
        resp, err := o.storage.ListVolumes(context.Background(), req)
	if err != nil {
		return nil, err
	}
        
        // process

        if resp.OpcNextPage == nil {
                  break
        }
        nextPage = resp.OpcNextPage
}

```